health-on-time/src/App.js

```diff
@@ -57,8 +59,14 @@ function App() {
         <Register handleLogin={handleLogin} BASE_URL={BASE_URL} />
       </Route>
       <Route path="/" component={HomePage} exact={true} />
-      <Route path="/medicine" component={MedicineCabinet} />
-      <Route path="/calendar" component={CalendarPage} />
+      <Route path="/medicine" component={Sidebar} />
+      <Route path="/calendar" component={Sidebar} />
+      {/* <Route
+        path="/app"
+        render={(props) => (
+          <Sidebar profileImgUrl="https://cdn.pixabay.com/photo/2016/08/20/05/38/avatar-1606916_960_720.png" />
+        )}
```

Comment:
I see this is where you were going to send props from App.js. You can structure the Route as I did with login and signup. 

<Route path="/app">
<Sidebar profileImageUrl={profile_pic}/>
</Route>
*** // profile_pic because that's the name of the URL's value when the state is updated in App.js. You can use react dev tools to see the state in app.js once user signs in to see what values you have access to.***